src/cloud-api-adaptor/Makefile

```diff
@@ -186,6 +187,10 @@ podvm-binaries:
 	--build-arg PODVM_DISTRO=$(PODVM_DISTRO) \
 	--build-arg ARCH=$(ARCH) \
 	--build-arg ATTESTER=$(ATTESTER) \
+	--build-arg PAUSE_REPO=$(PAUSE_REPO) \
```

Comment:
Any default value for PAUSE_REPO and PAUSE_VERSION?

Reply:
yeah, the default values are from version.yaml at here
https://github.com/confidential-containers/cloud-api-adaptor/blob/main/src/cloud-api-adaptor/Makefile.defaults#L44-L45
->  
https://github.com/confidential-containers/cloud-api-adaptor/blob/main/src/cloud-api-adaptor/versions.yaml#L49-L51